### DISCUSSION TITLE
Webhook for specific Content Type

### DISCUSSION THREAD

**Discussion by maximilianofried** [strapi/strapi#7730] [Legacy Topics]
Hello i want to know if there is a way to only trigger webhooks when a specific content type is created. Right now is working but the webhooks are trigger'd for all the content types i have in my collection. 

Thank you!

**Comment by MattieBelt** [comment]
Hey @maximilianofried, no this is currently not possible through the wehooks.
However the Lifecycle hooks can help you to achieve a webhook like behaviour.

You could for example make an `axios` request every time `afterUpdate` or `afterCreate` gets executed.
Check out [the documentation on Lifecycle hooks](https://strapi.io/documentation/v3.x/concepts/models.html#lifecycle-hooks).

**Reply by MattieBelt** [reply]
You also could take a look at the `run()` webhook function in the core, it's only using `fetch` instead of `axios`.
https://github.com/strapi/strapi/blob/69706d7587a50e48692bdb078fda46e1119a2cc8/packages/strapi/lib/services/webhook-runner.js#L77-L113
